docs/docusaurus/docs/reference/learn/data_quality_use_cases/schema_resources/schema_strict_and_relaxed.py

```diff
@@ -85,7 +85,7 @@
 
 relaxed_suite.add_expectation(
     gxe.ExpectColumnValuesToBeInTypeList(
-        column="transfer_amount", type_list=["DOUBLE_PRECISION", "STRING"]
+        column="transfer_amount", type_list=["DOUBLE PRECISION", "STRING"]
```

Comment:
@TrangPham This is a simple question (but just to confirm my understanding), but could you describe why this is changing from `DOUBLE_PRECISION` to `DOUBLE PRECISION`?

https://docs.sqlalchemy.org/en/20/dialects/postgresql.html#postgresql-data-types

Reply:
My understanding : SQALCHEMY calls it `DOUBLE_PRECISION` and POSTGRES actually calls it `DOUBLE PRECISION` (with no underscore). We are going with what postgres calls it.